src/allmydata/scripts/runner.py

```diff
@@ -41,7 +53,13 @@ class Options(usage.Options):
                     +   stats_gatherer.subCommands
                     +   admin.subCommands
                     + GROUP("Controlling a node")
-                    +   startstop_node.subCommands
+                    + [
+                        ["daemonize", None, tahoe_daemonize.DaemonizeOptions, "run a node disconnected from terminal"],
+                        ["start", None, tahoe_start.StartOptions, "start a node"],
+                        ["run", None, tahoe_run.RunOptions, "run a node"],
```

Comment:
It's not terribly clear what the difference between "start" and "run" is here.  I think the old text included the word "synchronously" which I don't think added a whole lot.  I think "run" is "run in the foreground", "start" is "old-style run daemonized, maybe deprecated soon", and "daemonize" is "new-style run daemonized"?